processing/src/main/java/org/apache/druid/jackson/DefaultObjectMapper.java

```diff
@@ -42,6 +42,8 @@
  */
 public class DefaultObjectMapper extends ObjectMapper
 {
+  public static final DefaultObjectMapper INSTANCE = new DefaultObjectMapper();
```

Comment:
The idea is that if a unit test needs the default mapper, then rather than creating one per test, and wondering which one we're using, we just use this single global one. Right now, there is only one test where I made that change. We can change others as we come across them.